_projects/git_task_list.md

```diff
@@ -0,0 +1,127 @@
+---
+collaborating_projects:
+  - gsoc-prep-tasks
+desc: "List open tasks from various issue tracker. Can be filtered to only show
+  newcomer friendly tasks."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - https://github.com/coala/projects/issues/480
```

Comment:
this was never part of your proposed work for this project, and is now being done by a different project. remove it, and instead add issues which your project will solve.